libs/java/auth_core/pom.xml

```diff
@@ -37,6 +37,11 @@
       <artifactId>libpam4j</artifactId>
       <version>1.8</version>
     </dependency>
+    <dependency>
+      <groupId>com.amazonaws</groupId>
+      <artifactId>aws-java-sdk-s3</artifactId>
+      <version>1.11.194</version>
```

Comment:
We should use ${aws.version} property from parent pom to have consistent aws library usage in Athenz.

Reply:
fixed